e2e/kube.go

```diff
@@ -527,31 +527,17 @@ func podHTTPServerLinux(s *Scenario) *corev1.Pod {
 	}
 }
 
-func podHTTPServerWindows(s *Scenario) *corev1.Pod {
+func podWindows(s *Scenario, podName string, imageName string) *corev1.Pod {
 	return &corev1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("%s-test-pod", s.Runtime.KubeNodeName),
+			Name:      fmt.Sprintf("%s-test-%s-pod", s.Runtime.KubeNodeName, podName),
 			Namespace: "default",
 		},
 		Spec: corev1.PodSpec{
 			Containers: []corev1.Container{
 				{
-					Name:  "iis-container",
-					Image: "mcr.microsoft.com/windows/servercore/iis",
-					Ports: []corev1.ContainerPort{
-						{
-							ContainerPort: 80,
-						},
-					},
-					ReadinessProbe: &corev1.Probe{
-						PeriodSeconds: 1,
-						ProbeHandler: corev1.ProbeHandler{
-							HTTPGet: &corev1.HTTPGetAction{
-								Path: "/",
-								Port: intstr.FromInt32(80),
-							},
-						},
-					},
```

Comment:
A minor thing. But I think readiness probe caught an issues before (maybe not).

I prefer to keep it, but I don't mind loosing it

Reply:
The readiness probe checks the container is responding to HTTP requests. Neither the servercore nor nanoserver containers have an HTTP server, so this readiness probe won't work anymore. 

If you have any ideas for a working readiness probe, that would be great!